files/hipchat.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+require "#{File.dirname(__FILE__)}/base"
+
+class Hipchat < BaseHandler
+  def handle
+    timeout_and_retry do
+      case @event['check']['status'].to_i
```

Comment:
aside: i don't think this really changes the behavior for hipchat_handler the way it did for pager duty. 

i made the change anyway for consistency.